src/index.js

```diff
@@ -467,7 +466,13 @@ function deleteFrame(){
   if(!confirm('This will delete the image from disk and remove it\'s tags from the save file.\nAre you sure you want to delete this image?')) return;
   let currFrameId = videotagging.getCurrentFrameId();
   
-  fs.unlinkSync(videotagging.imagelist[videotagging.imageIndex]);
+  try{
+    fs.unlinkSync(videotagging.imagelist[videotagging.imageIndex]);
```

Comment:
Shouldn't the rest of the function be in the try clause?  We want to abort everything if we can't delete the file on disk, right?